install.sh

```diff
@@ -167,6 +167,98 @@ mkdir -p "$OMS_CACHE/runtime-script"
 mkdir -p "$OMS_CACHE/startup-script"
 mkdir -p "$OMS_CACHE/trash"
 
+# install lolcat with apt-get if not found
+if [ ! -f "$OMS/lib/bin/lolcat" ]
+then
```

Comment:
Also, we should ask user if he really want to install lolcat.

Reply:
you have rewrite and fix all about or it's me I don't see ? haha

Reply:
Yep, I comment this before making changes.

Reply:
> Yep, I comment this before making changes.

So what do you mean now (the new pull request) ?